app/src/lib/git/core.ts

```diff
@@ -230,6 +231,17 @@ export async function git(
 
   log.error(errorMessage.join('\n'))
 
+  if (gitError && gitError === DugiteError.PushWithFileSizeExceedingLimit) {
```

Comment:
You don't really need the `gitError && ` part here, right?
```suggestion
  if (gitError === DugiteError.PushWithFileSizeExceedingLimit) {
```